src/main.cc

```diff
@@ -281,8 +290,8 @@ int main(int argc, char *argv[]) {
   evthread_use_pthreads();
 
   signal(SIGPIPE, SIG_IGN);
-  signal(SIGINT, signal_handler);
-  signal(SIGTERM, signal_handler);
+  signal(SIGINT, signalHandler);
+  signal(SIGTERM, signalHandler);
```

Comment:
I'm unsure whether this handle can be still registered properly. To me, the original `extern "C"` block is for C ABI compatibility.

Reply:
I re-added them.